### DISCUSSION TITLE
Duda con respecto a las practicas

### DISCUSSION THREAD

**Discussion by aldairxd04-garcia** [Launch-X-Latam/MisionFrontEnd#470] [Team Posts]
Me estoy poniendo al corriente con algunas sesiones, y escuche que el sherpa dijo que las practicas no eran obligatorias, pero para la constancia sí son obligatorias todas?

**Comment by Chemamx52** [comment]
Tengo la misma duda. También no se dónde se tienen que subir o colocar las practicas realizadas

**Comment by Rarriaza-sv** [comment]
https://github.com/orgs/Launch-X-Latam/teams/mision-frontend/discussions/40

**Comment by JoseEduardoVelazquezCN** [comment]
Según lo que publicaron recientemente de la misión backend habrá un tiempo para subir lo que hicimos, y muchos mencionan un formulario así que supongo que será al final de launch x

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by AlvaroFierro** [comment]
> Tengo la misma duda. También no se dónde se tienen que subir o colocar las practicas realizadas

En tu propio repositorio, en GitHub

**Comment by SirCesario** [comment]
Buenas Tardes,

Pero no entiendo la referencia , hace tiempo esperaba el inicio de las clases y hoy ya veo que estan montados unos repositorios de un Front completo y es como si estuviera atrasado un monton , en ese caso alguien sabe con quien se puede validar que se tendría que hablar para nivelar ?


Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows

De: ***@***.***>
Enviado: sábado, 5 de noviembre de 2022 5:00 p. m.
CC: ***@***.***>
Asunto: [Launch-X-Latam/mision-frontend] Duda con respecto a las practicas (#42)


Me estoy poniendo al corriente con algunas sesiones, y escuche que el sherpa dijo que las practicas no eran obligatorias, pero para la constancia sí son obligatorias todas?

**Comment by adrian-unmsm14** [comment]
Buenas noches, alguien revisará mi repositorio para verificar mis
prácticas??

**Comment by RafaelRuiz76** [comment]
Cómo le ago para servir mis prácticas guigaut

**Comment by Rarriaza-sv** [comment]
dejen de hacer spam a los demas , lean antes de escribir
https://github.com/orgs/Launch-X-Latam/teams/mision-frontend/discussions/40
